packages/build/src/core/normalize_flags.js

```diff
@@ -64,6 +65,7 @@ const computeTelemetry = function (flags, envOpts) {
 
 const REQUIRE_MODE = 'require'
 const DEFAULT_FUNCTIONS_DIST = '.netlify/functions/'
+const DEFAULT_BUILDERS_DIST = '.netlify/builders/'
```

Comment:
The flag defaults to `.netlify/builders/` in CLI builds.
In production, the buildbot will specify the value of the flag instead, setting it to some temporary directory like we do for regular functions.